website/docs/cloud-docs/vcs/github-app.mdx

```diff
@@ -60,6 +60,10 @@ HCP Terraform requests GitHub authorization from each user, displaying a pop-up
 
 ![Screenshot: GitHub asking whether you want to authorize "Terraform Cloud by HashiCorp".](/img/docs/gh-app-authorize.png)
 
+After initial configuration of GitHub App is performed, and therefore VCS provider instance created, GitHub App option will become grayed out and unavailable. You can modify existing configuration without removing it from GitHub, and re-creating.
```

Comment:
Can we move all of these sentences below, "Once you authorize the app, you can use GitHub in any of your HCP Terraform organizations without needing to re-authorize." since it corresponds with the screenshot above?

Reply:
This now sounds like a duplicate of changes you suggested. GitHub App configuration can be modified from both sides, added a complementary HCP side instead.